certbot-dns-bookmyname/certbot_dns_bookmyname/__init__.py

```diff
@@ -0,0 +1,87 @@
+"""
+The `~certbot_dns_bookmyname.dns_bookmyname` plugin automates the process of
+completing a ``dns-01`` challenge (`~acme.challenges.DNS01`) by creating, and
+subsequently removing, TXT records in a BookMyName domain zone.
+
+
+Named Arguments
+---------------
+
+==========================================  ===================================
+``--dns-bookmyname-credentials``            BookMyName credentials_ INI file.
+                                            (Required)
+``--dns-bookmyname-propagation-seconds``    The number of seconds to wait for
+                                            DNS to propagate before asking the
+                                            ACME server to verify the DNS
+                                            record.
+                                            (Default: 900)
+==========================================  ===================================
+
+
+Credentials
+-----------
+
+Use of this plugin requires a configuration file containing BookMyName
+credentials, which are the login and password of a BookMyName account
+allowed to modify the domain zone.
+
+.. code-block:: ini
+   :name: credentials.ini
+   :caption: Example credentials file:
+
+   # BookMyName API credentials used by Certbot
+   dns_bookmyname_login = SomeBookMyNameId
+   dns_bookmyname_password = CorrespondingPassword
+
+The path to this file can be provided interactively or using the
+``--dns-bookmyname-credentials`` command-line argument. Certbot records the
+path to this file for use during renewal, but does not store the file's contents.
+
+.. caution::
+   You should protect these API credentials as theyinclude the password to your
```

Comment:
nit-pick: "theyinclude" should be "they include"

Reply:
Will fix

Reply:
Done